D1673/P1673.md

```diff
@@ -3484,152 +3555,151 @@ void givens_rotation_setup(const complex<Real>& a,
                            complex<Real>& r);
 
 // [linalg.algs.blas1.givens.rot], apply computed Givens rotation
-template<class inout_vector_1_t,
-         class inout_vector_2_t,
+template<inout_vector InOutVec1,
+         inout_vector InOutVec2,
```

Comment:
Can you confirm that is how we want to spell it in code snippets (as compared to `@_inout-vector_@` which I assume would not be rendered properly)?
More particularly with an underscore rather than a dash

Reply:
This was a formatting bug.  I've fixed this in a comment I pushed today.  Thanks for pointing this out!